src/randoop/main/GenInputsAbstract.java

```diff
@@ -125,35 +124,6 @@ public GenInputsAbstract(String command, String pitch,
   @Option("Ignore class names specified by user that cannot be found")
   public static boolean silently_ignore_bad_class_names = false;
   
-
-  /**
-   * Use Randoop's default set of object contracts as assertions.
-   * If disabled, these assertions are not created.
-   * 
-   * <p>
-   * The default set of contracts includes:
-   *   equals(Object) is reflexive,
-   *   equals(Object) is symmetric,
-   *   equals(Object) and hashCode() are consistent,
-   *   x.equals(null) returns false,
-   *   any nullary method annotated with {@code @CheckRep} returns true.
-   * </p>
-   */
-  ///////////////////////////////////////////////////////////////////
-  @OptionGroup("Creating test oracles")
```

Comment:
because we removed the two options that were under it.
I'm not sure "oracles" makes sense anyway.

I did add a new group.